component/image/image.css

```diff
@@ -0,0 +1,26 @@
+@import '/index.css';
+
+.parent-container,
+.image-container {
+  display: flex;
+  margin: auto;
+  width: 50%;
+  height: 30rem;
+}
+
+.responsive-img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.image-container {
+  height: 20rem;
+  width: 20rem;
+}
+
+.rounded-img {
+  border-radius: 50%;
+  aspect-ratio: 1/1;
+  object-fit: cover;
```

Comment:
here you don't have to use object-fit cover, You can use responsive-img class with round-img class

Reply:
I have purposefully added it so user doesn't need to use different class😅